system/registry-cache/templates/deployment.yaml

```diff
@@ -38,6 +39,14 @@ spec:
 {{- end }}
         ports:
         - containerPort: {{ .Values.service.internalPort }}
+        livenessProbe:
+          exec:
+            command:
+              - sh
+              - -c
+              - md5sum /conf/tls.crt | cmp /tmp/tls.crt.md5
```

Comment:
have you tested that this works? are the `cmp` and `md5sum` binary available in the docker image?

Reply:
They are both part of busybox, i tested this in qa.